src/extension.ts

```diff
@@ -1,16 +1,16 @@
-import * as stripComments from 'strip-json-comments';
-
 import {
   ExtensionContext,
   Position,
   Range,
   TextEditorDecorationType,
   window,
-  commands
+  commands,
+  workspace
 } from 'vscode';
 
 import * as vscode from 'vscode';
 
+const stripComments = require('strip-json-comments');

```

Comment:
I was getting a warning that this module could not be imported. Changing from ES6 import to require made the warning go away.